examples/python/Misc/MultiDevice/ptp_frame_sync_minimal.py

```
@@ -0,0 +1,176 @@
+        deviceName = getDeviceName(device)
+
+        for socket in device.getConnectedCameras():
+            ########################################################################
+            # TODO: remove this when OV9282 is supporter for PTP
+            sensorName = ""
+            for sckt, sName in device.getCameraSensorNames().items():
+                if sckt == socket:
+                    sensorName = sName
+                    break
+            if sensorName == "":
+                raise RuntimeError(f"No sensor name found for {socket.name} on {deviceName}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)